x/muxer/muxer.go

```diff
@@ -127,16 +129,16 @@ func (m *muxer) CheckConfig(ctx context.Context, req *rpc.CheckRequest) (*rpc.Ch
 		// Add missing inputs, but don't override existing inputs.
 		for k, v := range r.A.GetInputs().GetFields() {
 			existingValue, has := inputs.Fields[k]
-			if has {
-				// If different servers return different check values,
-				// signal an error. We don't know how to deal with that.
-				if proto.Equal(existingValue, v) {
-					continue
-				}
-				return nil, fmt.Errorf("config value '%s' mismatch between servers %d and a previous server",
-					k, i)
+			if has && !proto.Equal(existingValue, v) {
+				// If different servers return different values, pick arbitrarily.
+				glog.V(9).Infof("[muxer] CheckConfig results do not agree on the '%s' property:"+
+					"\n    server %d: %s"+
+					"\n    server %d: %s"+
+					"\nPicking the server %d response",
```

Comment:
Likewise, having them numbered instead of named makes it difficult. These are the servers subject to muxing. I don't have better vocabulary at the moment but `glog.V(9)` is aimed at maintainers not users so it's not to the same level of scrutiny.